transcribe/requirements.txt

```diff
@@ -1,3 +1,4 @@
 openai-whisper
 tqdm
-librosa
\ No newline at end of file
+librosa
+ffmpeg-python
```

Comment:
`ffmpeg` needs to be install via `apt-get`, not pip. This package is not mandatory.

Reply:
yep i understood it now
in order for the test to pass we need to execute this command into the pipenv